docs/timelines/timeline-options.md

```diff
@@ -66,7 +66,7 @@ Timeline-Global Release Time | Sets the Global Release Time for playbacks trigge
 
 Option              | Action
 ---|-----
-Activate In Range | Off: Timeline must be fired manually. <br>**On:** Timeline will be fired automatically when timecode comes within the Start/End time range.
+Activate In Range | **Activate at 100%**: Timeline will be fired automatically at 100% level when timecode comes within the Start/End time range.<br>Activate at 0%: Will be fired automatically but at 0% level (zero intensity)<br>Off: Timeline will not be activated until manually fired.
```

Comment:
Might be covered somewhere else but I wonder if it's worth stressing that no triggers will be fired if the timeline is not activated.

Reply:
Added note in running-and-editing-timelines.md